server/src/test/java/org/elasticsearch/index/IndexingSlowLogTests.java

```diff
@@ -261,16 +278,16 @@ private void assertTimeValueException(final IllegalArgumentException e, final St
         assertThat(e.getCause(), instanceOf(IllegalArgumentException.class));
         final IllegalArgumentException cause = (IllegalArgumentException) e.getCause();
         final String causeExpected =
-                "failed to parse setting [" + key + "] with value [NOT A TIME VALUE] as a time value: unit is missing or unrecognized";
+            "failed to parse setting [" + key + "] with value [NOT A TIME VALUE] as a time value: unit is missing or unrecognized";
         assertThat(cause, hasToString(containsString(causeExpected)));
     }
 
     private IndexMetaData newIndexMeta(String name, Settings indexSettings) {
         Settings build = Settings.builder().put(IndexMetaData.SETTING_VERSION_CREATED, Version.CURRENT)
-            .put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, 1)
-            .put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, 1)
-            .put(indexSettings)
-            .build();
+                                 .put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, 1)
+                                 .put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, 1)
+                                 .put(indexSettings)
+                                 .build();
```

Comment:
That looks a bit odd to me and also unrelated to this change? Can you please restore the old formatting?

Reply:
I reverted back to previous formatting. must have done this by accident. sorry